### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kberg**
@bafolts I added a "questions" section at the bottom.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+We endeavor to have this app run on home Windows, Linux, Mac computers, and support modern popular browsers (sorry, IE11) so any changes that might seem browser / platform specific may get extra scrutiny.
+
+### Managing resource consumption
+
+This project is designed to run on single node servers or locally. Any change should consider the impact on the setup required for local users and the performance impact on a single node server. Your change might cause a significant consumption increase in memory, disk or CPU. You may be asked to make changes in response to that.
+
+## Questions?
+
+[Post an issue](https://github.com/bafolts/terraforming-mars/issues/new) or [Find us on Discord](https://discord.gg/AUfyx5ba)
```
Comment (vsrisuknimit) [inline CONTRIBUTING.md:47]: This specific Discord invite code that you created is valid for 24 h. Please use https://discord.gg/fWXE53K that is the same one of Readme.MD.

**Review by vsrisuknimit — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (kberg) [reply CONTRIBUTING.md:47]: Thanks!

**Comment by vsrisuknimit** [comment]
The discord link looks good. It sounds reasonable. But again I am not qualified to review anything haha. Tagging @bafolts to look at it.

**Review by vsrisuknimit — DISMISSED** [review]

**Comment by vsrisuknimit** [comment]
Wait I approved the one specific change or did I approve the whole thing. I am confused. Sorry.

**Comment by kberg** [comment]
> Wait I approved the one specific change or did I approve the whole thing. I am confused. Sorry.

Well, you did approve the whole thing. But I can wait. You can change your approval state if you don't want me to submit yet.

Or, just  go back to suggesting more changes.

**Comment by vsrisuknimit** [comment]
Let's wait for @bafolts .

**Review by bafolts — APPROVED** [review]
This looks good. :shipit:

**Pull request merged**